tests/manager/openAccount/managerCanOpenAccount.spec.js

```diff
@@ -1,31 +1,34 @@
 import { test } from '@playwright/test';
 import { faker } from '@faker-js/faker';
+import { AddCustomerPage } from '../../../src/pages/manager/AddCustomerPage';
+import { OpenAccountPage } from '../../../src/pages/manager/OpenAccountPage';
+
+let createdFirstName;
+let createdLastName;
 
 test.beforeEach( async ({ page }) => {
-  /* 
-  Pre-conditons:
-  1. Open Add Customer page
-  2. Fill the First Name.  
-  3. Fill the Last Name.
-  4. Fill the Postal Code.
-  5. Click [Add Customer].
-  6. Reload the page (This is a simplified step to close the popup).
-  */
+  const addCustomerPage = new AddCustomerPage(page);
+
+  createdFirstName = faker.person.firstName();
+  createdLastName = faker.person.lastName();
+  const postCode = faker.location.zipCode();
 
+  await addCustomerPage.open();
+  await addCustomerPage.fillFirstNameField(createdFirstName);
+  await addCustomerPage.fillLastNameField(createdLastName);
+  await addCustomerPage.fillPostCodeField(postCode);
+  await addCustomerPage.clickAddCustomerButton();
+  await addCustomerPage.reloadPage();
 });
 
 test('Assert manager can add new customer', async ({ page }) => {
-/* 
-Test:
-1. Click [Open Account].
-2. Select Customer name you just created.
-3. Select currency.
-4. Click [Process].
-5. Reload the page (This is a simplified step to close the popup).
-6. Click [Customers].
-7. Assert the customer row has the account number not empty.
+  const openAccountPage = new OpenAccountPage(page);
 
-Tips:
- 1. Do not rely on the customer row id for the step 13. Use the ".last()" locator to get the last row.
-*/
-});
\ No newline at end of file
+  await openAccountPage.open();
+  await openAccountPage.selectJustCreatedCustomer(createdFirstName, createdLastName);
+  await openAccountPage.selectValueFromCurrencyDropdown('Dollar');
+  await openAccountPage.clickProcessButton();
+  await openAccountPage.reloadPage();
+  await openAccountPage.clickCustomersButton();
+  await openAccountPage.assertAccountNumberOfCustomerIsNotEmpty('');
```

Comment:
```suggestion
  await openAccountPage.assertAccountNumberOfCustomerIsNotEmpty();
```

I believe this function should not have arguments